### PR TITLE
MAT-6240: Upgraded cqframework version to 3.2.0

### PR DESCRIPTION
**Pull request by gregory-akins**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-6240](https://jira.cms.gov/browse/MAT-6240)
(Optional) Related Tickets:

### Summary
Upgraded cqframework to 3.2.0.. lots of breaking changes; would appreciate some good eyes on this one.

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included.
* [X] No extemporaneous files are included (i.e Complied files or testing results).
* [X] This PR is merging into the **correct branch**.
* [X] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/mat/cql_elm_translation/cql_translator/TranslationResource.java [hunk 1]
```diff
@@ -112,19 +124,28 @@ public CqlTranslator buildTranslator(
...
+      //          CqlCompilerException.ErrorSeverity.Error,
+      //          signatureLevel,
+      //          options);
+
+      NamespaceInfo nsInfo = null;
+
+      libraryManager.setUcumService(ucumService);
+      libraryManager.getCqlCompilerOptions().setOptions(options);
+
+      return CqlTranslator.fromStream(nsInfo, sourceInfo, cqlStream, libraryManager);
```
Comment (RohitKandimalla) [inline src/main/java/gov/cms/mat/cql_elm_translation/cql_translator/TranslationResource.java:149]: Regarding setting `SignatureLevel` and `ErrorSeverity`, looks like `CqlCompilerOptions` has a constructor that accepts both along with other options. I was wondering if we can use them, Do we need them ? idk whats the importance of these configs. 

1. We can ask the Bryn's team and gather some knowledge on these configs.
2. We can push it as is, wait and see if there will be any Helpdesk tickets. Since we don't even how to test these configs, I am ok with going ahead with what you have.

   ```
 /**
     * Constructor with defined ErrorSeverity, SignatureLevel, and arbitrary number of options.
     *
     * @param errorLevel
     * @param signatureLevel
     * @param options
     */
    public CqlCompilerOptions(CqlCompilerException.ErrorSeverity errorLevel, LibraryBuilder.SignatureLevel signatureLevel, Options... options) {
        this.setOptions(options);
        this.errorLevel = errorLevel;
        this.signatureLevel = signatureLevel;
    }
```

**Review by RohitKandimalla — DISMISSED** [review]

**Review by RohitKandimalla — APPROVED** [review]
